src/verify_definitions.rs

```diff
@@ -0,0 +1,82 @@
+//! Verification of definitions
+//!
+//! Implement verification of definitions per the set.mm/iset.mm conventions.
+//! If the "exceptions" string is empty we use the "typical" set.mm values.
+//! The current typical values are "ax-*,df-bi,df-clab,df-cleq,df-clel".
+//! For glob syntax see: <https://docs.rs/globset/latest/globset/>
+//! but in the future we may reduce the glob language sophistication.
+//! For more information see:
+//! <https://us.metamath.org/mpeuni/conventions.html>
+//! <https://github.com/digama0/mmj2/blob/master/mmj2jar/macros/definitionCheck.js>
+//! and "Metamath: A Computer Language for Mathematical Proofs" by
+//! Norman Megill and David A. Wheeler, 2019, page 155.
+
+use crate::diag::Diagnostic;
+use crate::statement::StatementAddress;
+use crate::{Database, StatementRef, StatementType};
+use globset::{Glob, GlobSetBuilder};
+
+/// Verify the definition in this statement.
+/// Return a diagnostic if there's a problem, else None for no problem.
+fn verify_definition_statement(stmt: &StatementRef<'_>) -> Option<Diagnostic> {
+    // Rule 1: New definitions must be introduced using = or <->
+    // TODO
+    if let Ok(label) = std::str::from_utf8(stmt.label()) {
+        println!(" Need to check: $a statement {label}");
+    }
+
+    None
+}
+
+impl Database {
+    /// Verify that definitions meet set.mm/iset.mm conventions;
+    /// exclude *exceptions* from this check.
+    #[must_use]
+    pub fn verify_definitions(&self, exceptions: &str) -> Vec<(StatementAddress, Diagnostic)> {
+        let mut diags = vec![];
+        let exceptions_str = if exceptions.is_empty() {
+            "ax-*,df-bi,df-clab,df-cleq,df-clel" // Default for set.mm.
+        } else {
+            exceptions
+        };
```

Comment:
See comment above, this part is not required once you provide the default value for the command line argument (and was not going to be used since the argument was mandatory)

Reply:
also, you should not use the empty string as a signal of "not present". This is what `Option` is for.

Reply:
This is OBE, we now have a proper default (and the plan is to eventually eliminate globs entirely).